pkg/clientinfo/observer.go

```diff
@@ -1,4 +1,4 @@
-package metrics
+package clientinfo
```

Comment:
> Should we rename Observer to MetricsObserver?

We could do that.

Reply:
[86dacb6](https://github.com/keep-network/keep-common/pull/105/commits/86dacb6a965501d8ba579361d0cf01941dc62b7c)